src/main/java/sirius/web/util/BarcodeController.java

```diff
@@ -49,6 +50,16 @@ public void qr(WebContext ctx) throws Exception {
         barcode(ctx, BarcodeFormat.QR_CODE);
     }
 
+    /**
+     * Creates a barcode for the given content.
+     * <p>
+     * The parameter <tt>content</tt> determines the contents of the barcode. The parameters <tt>width</tt> and
+     * <tt>height</tt> determine its dimensions.
+     * </p>
+     *
```

Comment:
```suggestion
     * <tt>height</tt> determine its dimensions.
     *
```

Reply:
Its crazy but JavaDoc is no real HTML syntax. So they use HTML tags to do some stuff, but for example Paragraph Tags should only open and not be closed ...

Reply:
Thank you, that closing tag slipped through my cleanup :)